exercicios/para-casa/entregas/exemplo-nome-sobrenome/emilly-santos/Pix.js

```diff
@@ -0,0 +1,46 @@
+const { Conta } = require('./ContaModificada');
+
+class Pix {
+    #saldo;
+
+    constructor(saldo) {
+        this.#saldo = saldo;
+    }
+
+    criarChavePix(chavePix, tipo) {
+        const regexMap = {
```

Comment:
muito legal a forma como você desenvolveu a variável regexMap, esse tipo de construção chama Dispatch Table (caso queira dar uma lida: [https://dev.to/philiphaines/dynamic-dispatch-and-dispatch-tables-kdm](url))
Parabéns 🎉